lib/generators/devise_security/install_generator.rb

```diff
@@ -11,8 +11,8 @@ class InstallGenerator < Rails::Generators::Base
       desc 'Install the devise security extension'
 
       def copy_initializer
-        template('devise-security.rb',
```

Comment:
We will also need to support the non-snake-case version because:

1) Support for existing codebases so that this isn't a breaking change.
2) For anyone who disagrees with the cop and wants the dash instead of the underscore.

Reply:
@oniofchaos by support, are you thinking we should be able to generate both filenames? 

From my understanding of how things work, it seems like the file name doesn't end up really mattering. I believe Rails will load all files in the `config/initializers` directory and I don't think the code has any direct reference to the `devise-security.rb`, we just expect the config file to leverage `Devise.setup`:

https://github.com/devise-security/devise-security/blob/ad50cdb888473c38b4023c2343f6c014604f5b84/lib/generators/templates/devise-security.rb#L3

Reply:
That's a very good point. I misread what some of this code was doing.

One thing I'd like to see before this is merged is an example app that has the "old" config name and uses this branch of the gem and demos that it still works.